### PR TITLE
Milestone/data bridge migration

### PR DESCRIPTION
**Pull request by tomaszmichalak**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -37,5 +34,27 @@ subprojects { subproject ->
 
     compileJava compileOptions
     compileTestJava compileOptions
+
+    dependencies {
+      compileOnly group: 'io.vertx', name: 'vertx-codegen', version: '3.5.0'
```
Comment (marcinczeczko) [inline build.gradle]: use knotx-dependencies pom here to get right versions. see how to do it https://docs.gradle.org/4.6/userguide/managing_transitive_dependencies.html?_ga=2.63949359.1281256360.1523425727-158124660.1522940345#sec:bom_import

File: build.gradle [hunk 2]
```diff
@@ -37,5 +34,27 @@ subprojects { subproject ->
...
+      testCompile group: 'io.vertx', name: 'vertx-unit', version: '3.5.0'
+      testCompile group: 'junit', name: 'junit', version: '4.12'
+      testCompile group: 'org.powermock', name: 'powermock-api-mockito', version: '1.6.6'
+      testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '1.6.6'
+      testCompile group: 'org.mockito', name: 'mockito-all', version: '2.0.2-beta'
+
+      if (subproject.path.startsWith(":adapter:http")) {
+        compile project(':adapter:api')
+      }
+
```
Comment (marcinczeczko) [inline build.gradle]: We need also added support for releasing to maven central. see sonatype documenation how to do it

File: data-source/http-stale/build.gradle [hunk 3]
```diff
@@ -0,0 +1,25 @@
+plugins {
```
Comment (marcinczeczko) [inline data-source/http-stale/build.gradle]: what `http-stale` means ??

**Comment by marcinczeczko** [comment]
What about /config folder with bridge.conf files ?

File: build.gradle [hunk 4]
```diff
@@ -4,20 +4,17 @@
...
 
 allprojects {
   group = 'io.knotx.databridge'
+  version = '1.3.0-SNAPSHOT'
 
   apply plugin: "eclipse"
   apply plugin: "idea"
 
   repositories {
+    mavenLocal()
```
Comment (pun-ky) [inline build.gradle]: I recommend to remove it / to prevent "on my machine works"

File: build.gradle [hunk 5]
```diff
@@ -37,5 +34,27 @@ subprojects { subproject ->
 
     compileJava compileOptions
     compileTestJava compileOptions
+
+    dependencies {
+      compileOnly group: 'io.vertx', name: 'vertx-codegen', version: '3.5.0'
+      compileOnly group: 'io.knotx', name: 'knotx-core', version: '1.3.0-SNAPSHOT'
+
+      testCompile group: 'io.knotx', name: 'knotx-core', version: '1.3.0-SNAPSHOT'
```
Comment (pun-ky) [inline build.gradle]: of course after having "no snapshot" version

**Pull request merged**